src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -7,19 +7,27 @@
 
 namespace SprykerSdk\Spryk\Console;
 
+use Symfony\Component\Console\Helper\FormatterHelper;
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SprykDumpConsole extends AbstractSprykConsole
 {
+    protected const COMMAND_NAME = 'spryk:dump';
+    protected const COMMAND_DESCRIPTION = 'Dumps a list of all Spryk definitions.';
+
+    public const ARGUMENT_SPRYK = 'spryk';
```

Comment:
It's only needed to be public to be used in tests? I'd rather have a duplicate in tests instead of making this public. This is not a blocker.

Reply:
`ARGUMENT_SPRYK`  of `SprykRunConsole` is used in 32 different test, I would keep it for consistency or refactor all these later.